controllers/suite_test.go

```diff
@@ -95,9 +94,19 @@ var _ = BeforeSuite(func() {
 	ManifestPath = strings.Replace(ManifestPath, ".", "..", 1) // This is needed as the tests need to reference a directory backward
 
 	err = (&MetalLBReconciler{
-		Client: k8sClient,
-		Scheme: scheme.Scheme,
-		Log:    ctrl.Log.WithName("controllers").WithName("MetalLB"),
+		Client:    k8sClient,
+		Scheme:    scheme.Scheme,
+		Log:       ctrl.Log.WithName("controllers").WithName("MetalLB"),
+		Namespace: MetalLBTestNameSpace,
+	}).SetupWithManager(k8sManager)
+	Expect(err).ToNot(HaveOccurred())
+
+	AddressPoolManifestPath = strings.Replace(AddressPoolManifestPath, ".", "..", 1) // This is needed as the tests need to reference a directory backward
```

Comment:
Just assign it the full path as string, I think it's more readable

Reply:
Also, restore the original path in AfterSuite.
It's not the case here (now), but there could be other tests relying on the original path